src/main/scala/shine/OpenCL/Compilation/Passes/InsertMemoryBarriers.scala

```diff
@@ -17,27 +17,25 @@ object InsertMemoryBarriers {
   def insert: Phrase[CommType] => Phrase[CommType] = p =>
     analyzeAndInsertBarriers(p, Map())._1
 
-  private case class Metadata(// reads from outer scope allocations
-                              reads: mutable.Map[Identifier[_ <: PhraseType], AddressSpace],
-                              // work-group parallel writes to outer scope allocations
-                              wg_writes: mutable.Map[Identifier[_ <: PhraseType], AddressSpace])
+  private case class Metadata( // reads from outer scope allocations
+                               reads: mutable.Map[Identifier[_ <: PhraseType], AddressSpace],
+                               // work-group parallel writes to outer scope allocations
+                               wg_writes: mutable.Map[Identifier[_ <: PhraseType], AddressSpace])
 
-  private def analyzeAndInsertBarriers(
-    p: Phrase[CommType],
-    // allocations in the current scope
-    allocs: Map[Identifier[_ <: PhraseType], AddressSpace]
-  ): (Phrase[CommType], Metadata) = {
+  private def analyzeAndInsertBarriers(p: Phrase[CommType],
+                                       // allocations in the current scope
+                                       allocs: Map[Identifier[_ <: PhraseType], AddressSpace]
+                                      ): (Phrase[CommType], Metadata) = {
     val meta = Metadata(mutable.Map(), mutable.Map())
     val p2 = VisitAndRebuild(p, Visitor(allocs, meta))
     (p2, meta)
   }
 
-  private def visitLoopBody(
-    p: Phrase[CommType],
-    allocs: Map[Identifier[_ <: PhraseType], AddressSpace],
-    metadata: Metadata,
-    outer_wg_writes: mutable.Map[Identifier[_ <: PhraseType], AddressSpace] = mutable.Map()
-  ): Phrase[CommType] = {
+  private def visitLoopBody(p: Phrase[CommType],
+                            allocs: Map[Identifier[_ <: PhraseType], AddressSpace],
+                            metadata: Metadata,
+                            outer_wg_writes: mutable.Map[Identifier[_ <: PhraseType], AddressSpace] = mutable.Map()
```

Comment:
Yeah, I know that you are not a fan of this style, but it's pretty much everywhere.
I will check if the lines are longer than 100 chars, otherwise, I will keep it.